NebulaHost/PacketProcessors/Factory/PasteEntitySettingUpdateProcessor.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NebulaModel.Attributes;
+using NebulaModel.Networking;
+using NebulaModel.Packets.Factory;
+using NebulaModel.Packets.Processors;
+using NebulaWorld.Factory;
+
+namespace NebulaHost.PacketProcessors.Factory
+{
+    [RegisterPacketProcessor]
+    class PasteEntitySettingUpdateProcessor : IPacketProcessor<PasteEntitySettingUpdate>
+    {
+        public void ProcessPacket(PasteEntitySettingUpdate packet, NebulaConnection conn)
+        {
+            EntitySettingDesc backup = EntitySettingDesc.clipboard;
+            EntitySettingDesc.clipboard = packet.GetEntitySettings();
+            FactoryManager.EventFromClient = true;
+            GameMain.localPlanet.factory.PasteEntitySetting(packet.EntityId);
```

Comment:
This should also get the factory from the packet since we could be on another planet completely

Reply:
right, I will fix it :)